.github/PULL_REQUEST_TEMPLATE/dependency-change.md

```
@@ -0,0 +1,71 @@
+## Dependency change summary
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)